### PR TITLE
More UI & Game Reset

### PR DESCRIPTION
**Pull request by kavanparam**
- Grid is initialized with useEffect hook
- GameBoard blocks are responsive
- Added buttons to display game rules & reset `grid` state (New Game & Game Rules)
- New Game button is functional & resets the GameBoard grid

image

### CODE REVIEW

File: pages/studentPortal/labs/multiplication-connect/Index.tsx [hunk 1]
```diff
@@ -87,16 +69,60 @@ export const calculateWinner = (grid: GameBoardBlock[]) => {
...
+      });
+    }
+    console.log("Grid created");
+    console.log(newGrid);
+    return newGrid;
+  };
+
+  useEffect(() => {
+    console.log(`newGame: ${newGame}`);
+    setGrid(createGrid);
```
Comment (vithushan19) [inline pages/studentPortal/labs/multiplication-connect/Index.tsx:95]: I think your effect is set up correctly. But should you be invoking the createGrid function here?

createGrid()

File: pages/index.tsx [hunk 2]
```diff
@@ -21,10 +21,10 @@ const HomePage = () => {
     return () => window.removeEventListener("scroll", onScroll);
   }, []);
 
-  const minoritiesCopy: LandingPageCopy = {
+  const defaultCopy: LandingPageCopy = {
```
Comment (vithushan19) [inline pages/index.tsx:24]: This was my change. you should merge latest main into your branch.

File: pages/studentPortal/labs/multiplication-connect/Index.tsx (hunk 1 above)
Comment (kavanparam) [reply pages/studentPortal/labs/multiplication-connect/Index.tsx:95]: I've tried this but it isn't working as intended. Could you help with debugging it when you can?

Comment (vithushan19) [reply pages/studentPortal/labs/multiplication-connect/Index.tsx:95]: I won't be able to. Try booking Abhi or Jason

**Pull request merged**